spec/rubocop/cop/rails/no_variable_precision_decimal_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::NoVariablePrecisionDecimal, :config do
+  subject(:cop) { described_class.new(config) }
+
+  let(:cop_config) { { 'Include' => nil } }
+
+  context 'with add_column call' do
+    context 'with precision missing' do
+      it 'reports an offense' do
+        expect_offense(<<~RUBY)
+          add_column :fees, :amount, :decimal, scale: 4
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Do not define a Decimal column without explicit scale and precision parameters.
+        RUBY
+      end
+    end
+
+    context 'with scale missing' do
+      it 'reports an offense' do
+        expect_offense(<<~RUBY)
+          add_column :fees, :amount, :decimal, precision: 6
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Do not define a Decimal column without explicit scale and precision parameters.
```

Comment:
I would avoid negatives: "Set explicit scale and precision when adding Decimal column."

Reply:
Fixed